ReSharper.FSharp/src/FSharp.Psi/src/Impl/Tree/MemberDeclaration.cs

```diff
@@ -66,7 +65,9 @@ private IDeclaredElement CreateProperty(FSharpMemberOrFunctionOrValue mfv)
     {
       foreach (var accessor in AccessorDeclarationsEnumerable)
         if (accessor.IsExplicit)
-          return new FSharpPropertyWithExplicitAccessors(this);
+          return SourceName == StandardMemberNames.DefaultIndexerName && !IsStatic
```

Comment:
we can add the IsIndexer flag to the IMemberDeclaration

Reply:
Yup, sounds reasonable, but it's OK if done later.

I'd probably also split member declarations to be different types for methods and properties.